.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+# Description of PR purpose/changes
+
+* Please include a summary of the change and which issue is fixed. 
+* Please also include relevant motivation and context.
+* List any dependencies that are required for this change.
+
+## Jira Ticket / Issue #
+e.g. https://kbase-jira.atlassian.net/browse/DATAUP-X
+
+# Testing Instructions
+* Details for how to test the PR: 
+- [ ] Tests pass in travis and locally
+- [ ] Changes available by spinning up a local narrative and navigating to _X_ to see _Y_
+
+# Dev Checklist:
+
+- [ ] My code follows the guidelines at https://sites.google.com/truss.works/kbasetruss/development
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] New and existing unit tests pass locally with my changes
+- [ ] Any dependent changes have been merged and published in downstream modules
+- [ ] Title of the PR contains Jira ticket followed by the description of the PR e.g. (DATAUP-69 Adds a PR template)
```

Comment:
I think we can remove this item

Reply:
I don't know which item you are talking about, but I think you mean the jira one. The github UI doesn't make it clear. You need the ticket number in the title of the PR for the JIRA automation to work.